spec.md

```diff
@@ -232,12 +233,14 @@ Policies MUST specify the following fields:
   for this domain might be handled by any MX whose hostname is a subdomain of
   "example.com" or "example.net."
 * a: The mechanism to use to authenticate this policy itself. See the section
-  _Policy_ _Authentication_ for more details. Possible values are:
-  * webpki:URI, where URI points to an HTTPS resource at the recipient domain
-    that serves the same policy text.
+  _Policy_ _Discovery_ _&_ _Authentication_ for more details. Possible values are:
+  * webpki: Indicating that the policy is expected to be served over HTTPS on a
+    well-known URI: https://policy._smtp_sts.example.com/current. The 
+    example.com is the recipient's domain, and the endpoint serves the same policy
+    text.
   * dnssec: Indicating that the policy is expected to be served over DNSSEC.
 * c: Constraints on the recipient MX's TLS certificate (plain-text, required).
-  See the section _Policy_ _Validation_ for more details. Possible values are:
+  See the section _Recipient's_ _Certificate_ _Validation_ for more details. Possible values are:
```

Comment:
Good point. Agreed on this. 

I was only bike-shedding about "authentication" and "validation" because we had that long talk about the different stages/semantics at MAAWG, so I was trying to hew to that terminology. ;)